packages/panels/src/Pages/Auth/Register.php

```diff
@@ -77,9 +81,23 @@ public function register(): ?RegistrationResponse
         }
 
         $user = $this->wrapInDatabaseTransaction(function () {
+            $this->callHook('beforeValidate');
+
             $data = $this->form->getState();
 
-            return $this->handleRegistration($data);
+            $this->callHook('afterValidate');
+
+            $data = $this->mutateFormDataBeforeCreate($data);
+
+            $this->callHook('beforeCreate');
+
+            $this->record = $this->handleRegistration($data);
+
+            $this->form->model($this->record)->saveRelationships();
```

Comment:
Shouldn't this also be done in the handleRegistration method?

Reply:
This is consistent with the create page